supermarq-benchmarks/supermarq/qcvv/base_experiment.py

```diff
@@ -0,0 +1,512 @@
+# Copyright 2021 The Cirq Developers
```

Comment:
do we need the cirq license on these?

Reply:
I was copying what seemed to be the pattern to have Cirq headers wherever we import cirq. If thats not the case then we can remove

Reply:
sounds good. we can leave it for now, and sort it out w/ #983